.github/workflows/build.yml

```diff
@@ -221,44 +221,40 @@ jobs:
             docker.io/hashicorp/${{ env.repo }}-control-plane:${{ env.version }}
             986891699432.dkr.ecr.us-east-1.amazonaws.com/hashicorp/${{ env.repo }}-control-plane:${{ env.version }}
           dev_tags: |
-            hashicorppreview/${{ env.repo }}-control-plane:${{ env.version }}-dev
+            hashicorppreview/${{ env.repo }}-control-plane:${{ env.version }}
```

Comment:
The dev version already has the `-dev` suffix and so our image before ended up having `-dev-dev` suffix